server/src/main/java/dev/slimevr/Keybinding.java

```diff
@@ -69,6 +75,16 @@ public void onHotKey(int identifier) {
 				LogManager.info("[Keybinding] Reset mounting pressed");
 				server.scheduleResetTrackersMounting(this.config.getResetMountingDelay());
 			}
+			case FEEDBACK_SOUND -> {
```

Comment:
Wait what is going on here, why is there a keybind to play the sound?

Reply:
This is the keybind for enable or disable the feedback sound for tap only

Reply:
That's a weird feature 😅 It should be done via settings, not keybind.

Reply:
We have that toggle with setting, make sense to me for removing the toggle in keybind